package.json

```diff
@@ -23,16 +23,16 @@
 		"compile:common": "tsc --project ./tsconfig.build.json --outDir ./dist/cjs --module CommonJS",
 		"compile:esm": "tsc --project ./tsconfig.build.json --outDir ./dist/esm",
 		"lint": "eslint .",
-		"prepush:lint": "lint-staged",
-		"prepush:test": "jest --verbose --runInBand --onlyChanged",
+		"precommit:lint": "lint-staged",
+		"precommit:test": "jest --verbose --runInBand --onlyChanged",
```

Comment:
do we want to run tests on pre-commit? maybe these should stay pre-push?

Reply:
You think so? I figure if the test fails you've got to fix it, which is another code change. And I'd rather commit code that passes the test. But it does hurt ones ability to check-in partially completed work. I could be  persuaded either way on this.

Reply:
fair enough, let's try it and see!